recipes/configure.rb

```diff
@@ -1,29 +1,29 @@
 #
-# Cookbook:: nexus-iq-server
+# Cookbook:: nexus_iq_server
 # Recipe:: configure
 #
 # Copyright:: Copyright (c) 2017-present Sonatype, Inc. Apache License, Version 2.0.
 
-config_path = node['nexus-iq-server']['conf_dir'] + '/config.yml'
-start_script = node['nexus-iq-server']['install_dir'] + '/start-nexus-iq-server.sh'
```

Comment:
start-nexus-iq-server.sh is ok

Reply:
Changed to use hyphens for file system names.